### DISCUSSION TITLE
How to start the red lora icon?

### DISCUSSION THREAD

**Discussion by Drgoat1111** [d8ahazard/sd_dreambooth_extension#989] [Q&A]
![image](https://user-images.githubusercontent.com/126750891/222343974-4f353a0b-afea-4598-8e8b-6babe0eb772f.png)
![image](https://user-images.githubusercontent.com/126750891/222344096-9e854cde-b153-47d8-924c-af356cdcd362.png)
How to start the red lora icon? The first picture is someone else's. The second picture does not have this icon after I installed it, but my classmate has this icon as soon as the picture is installed. Can anyone help me

**Comment by cerega66** [comment]
You have an old version of web ui. Update it.
